src/lib.rs

```diff
@@ -237,7 +237,7 @@ use std::time::Duration;
 use curl::easy::Easy;
 use rand::{thread_rng, Rng};
 use serde::de::DeserializeOwned;
-use serde::Serialize;
+use serde::{Serialize};
```

Comment:
nit: let's remove the curly braces

actually, mind running `cargo +nightly fmt`? there are a few minor formatting issues that I think will be fixed by that command